api-docs/swagger/v1/swagger.json

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Should this file be a `.yml`?

Reply:
But it is a `yaml` file, so it should have the `.yml` extension.

Reply:
agreed let me do these changes and then let see how it goes

Reply:
extensions updated